enterprise_learner_portal/utils.py

```diff
@@ -44,3 +47,40 @@ def get_course_run_status(course_overview, certificate_info, enterprise_enrollme
     if course_overview['has_started']:
         return CourseRunProgressStatuses.IN_PROGRESS
     return CourseRunProgressStatuses.UPCOMING
+
+
+def get_exec_ed_course_run_status(course_details, certificate_info, enterprise_enrollment):
+    """
+    Get the status of a exec ed course run, given the state of a user's certificate in the course.
+
+    A run is considered "complete" when either the course run has ended OR the user has earned a
```

Comment:
From what I understand, the course run attached to an exec ed course is just a placeholder and may not have correct dates ... can we check with the Phoenix team if dates on Exec-Ed courses are usable?

Reply:
I confirmed from Phoenix team that the course_run dates for ExecEd courses has been deprecated, they are getting course dates through the [AdditionalMetadata model](https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/course_metadata/models.py#L834-L924)  in the discovery that we are importing in our CourseDetails model (our new source of truth).